dev-env/windows/manifests/vcredist-14.24.28127.4.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
To add some context for @aherrmann-da and others: The official Azure pipelines seem to have a newer version of this and if we try to run the binary built there on our own nodes it fails with a missing DLL error.

Reply:
That reminds me, I have not yet tested whether we need both or just the 64bit one.

Reply:
Getting closer to scoop upstream seems like a good idea anyway so I wouldn’t bother debugging this.

Reply:
👍